build_related_scripts/get_modified_packs.py

```diff
@@ -28,7 +28,7 @@ def option_handler() -> argparse.Namespace:
     """
     parser = argparse.ArgumentParser(description='Collect the packs that has changed.')
     parser.add_argument('-rp', '--repo_path', help='The path to the required repo.', type=dir_path)
-    parser.add_argument('--prev-ver', default='main', help='Previous branch or SHA1 commit to run checks against.')
+    parser.add_argument('--prev-ver', default='master', help='Previous branch or SHA1 commit to run checks against.')
```

Comment:
@ChanochShayner , thanks! I'm also changing this. While GitHub defaults to main now, this project still uses master. So I'm sticking with master for now. This at least lets other orgs who cannot fork and use main as the default to copy/paste and only have to supply the flag in the workflow file.

Reply:
The alternative is to modify config.yml's arguments, but since we're changing nothing else of it here, I tried to keep things simpler.